packages/admin-ui/package.json

```diff
@@ -91,8 +91,7 @@
     "nightwatch": "nightwatch --config ./src/tests/Nightwatch/nightwatch.conf.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "// Test commands for the CI": "⬅✌➡",
-    "test": "yarn test:unit",
-    "test:functional": "yarn nightwatch",
```

Comment:
It feels like it would be nice to be able to run unit tests independent from really slower functional tests

Reply:
@dawehner the unit tests are so fast I don't think it's worth splitting into separate jobs currently

Reply:
That makes sense, thank you

Reply:
I think this PR is good improvement.

I have tried the caution approach of nuking my setup and starting from scratch, and then pig headily trying all the commands.
 
a) I about to file a separate issue but

 yarn workspace @drupal/admin-ui build

failed  -  reporting that a script could not run 'react-scripts'  the solution to this was to run yarn install - and ensure it was available. We have just had a bump in react-scripts version number committed. So something for me to look into elsewhere.

b) With a "I'm trying to break things hat on".. can I ask for a minor touch up to the text of readme.md

there seem to be two candidate locations for .env.local
in the root directory "admin-ui"

or 

in the package directory "admin-ui/packages/admin-ui"

The text does not say which.

I think if we "time and motioned" 100 developers we would see confusion on this point..., Maybe I am projecting.

c) Lastly I never have or can now get the nightwatch tests to pass.  I see error messages to the effect of looking for visually hidden element but element not found  - timed out.

So what I am really saying is this PR is almost ready ... If I ever get to the bottom on my problems and it is helpful to the wider community I will create a new issue - but I did not want to given the PR a clean bill of health with that caveat.